CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## unreleased
+                                
+* TODO release 2.0.0 - describe changes better
+* gRPC not supported anymore
```

Comment:
Let's mention that the starter only supports sending data via HTTP in the README too.